moped-editor/src/queries/project.js

```diff
@@ -337,11 +337,22 @@ export const TIMELINE_QUERY = gql`
   }
 `;
 
+/**
+ * For all the following project phase mutations, the mutation to set
+ * current phases to false must run first or unique constraint could be violated
+ */
+
 export const ADD_PROJECT_PHASE = gql`
   mutation AddProjectPhase(
     $objects: [moped_proj_phases_insert_input!]!
     $current_phase_ids_to_clear: [Int!] = []
   ) {
+    update_moped_proj_phases(
```

Comment:
moving this operation to the top ensures that we are first updating the phase that is already current to not be current before we update the new phase to be current, so that we dont violate the unique index

Reply:
glad this worked out - thanks for verifying! 🙌